packages/core/src/database/index.ts

```diff
@@ -141,67 +141,69 @@ export const createDatabase = (args: {
   ////////
 
   let dialect: Database["dialect"];
-  let driver: Database["driver"];
+  let driver!: Database["driver"];
   let qb: Database["qb"];
 
   if (args.databaseConfig.kind === "pglite") {
     dialect = "pglite";
     namespace = "public";
 
-    const userFile = path.join(args.databaseConfig.directory, "public.db");
-    const syncFile = path.join(args.databaseConfig.directory, "ponder_sync.db");
+    const poolConfig: PoolConfig = {
+      max: 20,
+    };
 
-    // driver = {
-    //   user: createSqliteDatabase(userFile),
-    //   readonly: createReadonlySqliteDatabase(userFile),
-    //   sync: createSqliteDatabase(syncFile),
-    // };
+    driver = {
+      internal: createPool(poolConfig),
+      user: createPool(poolConfig),
+      readonly: createReadonlyPool(poolConfig),
+      sync: createPool(poolConfig),
+    };
 
     qb = {
       internal: new HeadlessKysely({
         name: "internal",
         common: args.common,
-        dialect: new SqliteDialect({ database: driver.user }),
+        dialect: new PostgresDialect({ pool: driver.internal }),
```

Comment:
Now, when we create the query builder (Kysely) objects, we can use the `kysely-pglite` driver (https://github.com/dnlsandiego/kysely-pglite/tree/main).

We should use the `user` driver for the `internal`, `user`, and `readonly` qb objects. I think this `qb` object should look identical to the one below for postgres except for the `dialect` option.